owrx/audio/wav.py

```diff
@@ -85,7 +85,9 @@ def switchFiles(self):
             file = self.wavefile
             self.wavefile = self.getWaveFile()
 
-        file.close()
+        if file != None:
```

Comment:
The python style guide recommends checking for `None` values by means of the `is` operator: https://peps.python.org/pep-0008/#programming-recommendations

`if file is not None:`

Reply:
Not a problem, will make the change.